src/components/select/template.njk

```diff
@@ -0,0 +1,19 @@
+{% from "label/macro.njk" import govukLabel %}
+{% set labelArguments = {
+  text: params.label.text,
+  html: params.label.html,
+  hintText: params.label.hintText,
+  hintHtml: params.label.hintHtml,
+  for: params.id,
+  errorMessage: params.errorMessage
+} %}
+{%- if params.label %}
+{{- govukLabel(labelArguments) -}}
```

Comment:
Not sure there's much benefit to defining the labelArguments outside of the call, given they're not used anywhere else.

You're also omitting some arguments from the `label` component - `classes` and `attributes` – as per https://github.com/alphagov/govuk-frontend/blob/master/src/components/file-upload/template.njk#L5-L14

Reply:
goos shout, fixed